src/components/Radio/Radio.tsx

```diff
@@ -143,7 +146,11 @@ export const Radio = React.forwardRef<HTMLInputElement, RadioProps>(
     };
 
     return (
-      <div className={clsx(classes.root, className)}>
+      <div
+        role="radio"
+        className={clsx(classes.root, className)}
+        onClick={() => inputRef?.current?.click()}
```

Comment:
Does this `onClick` sync to the onChange call in the `<input` tag through the `inputRef`?

Reply:
Yup, it clicks the input element which triggers whatever callbacks the input has when clicked. So clicking this parent div is equivalent to clicking the input itself. 

Its common to have a setup like this for file inputs when you don't want to use the native file input display so you hide the file input and click it via ref when some other element is clicked.